src/encoded/static/components/search.js

```diff
@@ -452,11 +451,6 @@ const DatasetComponent = (props, reactContext) => {
                         <div><strong>Project: </strong>{result.award.project}</div>
                     </div>
                 </div>
-                {activeCart ?
-                    <div className="result-item__cart-control">
-                        <CartToggle element={result} />
```

Comment:
I think you can take out CartToggle at the top because this is the only place I see you including that component.

Reply:
That’s weird. I already removed CartToggle from the top, but for some reason this PR still shows it.

Reply:
Oh wait, that's not right. I’m being confused.

Reply:
OK, I’m straightened out. CartToggle is still used in ExperimentComponent in that file. I just removed it from DatasetComponent.

Reply:
oh, I see, that's right.